zuul.d/nodeset.yaml

```diff
@@ -44,31 +44,12 @@
         label: centos-9-stream-crc-2-30-0-xxl
 
 - nodeset:
-    name: centos-9-rhel-9-2-crc-extracted-2.30-3xl
+    name: centos-9-multinode-rhel-9-2-crc-extracted-2-36-0-3xl-novacells
     nodes:
       - name: controller
         label: cloud-centos-9-stream-tripleo
       - name: crc
-        label: coreos-crc-extracted-2-30-0-3xl
-      - name: standalone
-        label: cloud-rhel-9-2-tripleo
-    groups:
-      - name: computes
-        nodes: []
-      - name: ocps
-        nodes:
-          - crc
-      - name: rh-subscription
-        nodes:
-          - standalone
-
-- nodeset:
-    name: centos-9-multinode-rhel-9-2-crc-extracted-2.30-3xl-novacells
-    nodes:
-      - name: controller
-        label: cloud-centos-9-stream-tripleo
-      - name: crc
-        label: coreos-crc-extracted-2-30-0-3xl
+        label: coreos-crc-extracted-2-36-0-3xl
```

Comment:
@bogdando i spotted this while doing a tidy up of our nodets to remove the 'old' 2.30 crc ones

the novacells is still on 2.30 so I updated it here.

Reply:
I'll wait for Bogdan to reply - pinged on https://github.com/openstack-k8s-operators/ci-framework/pull/1984#pullrequestreview-2168298204 - might be we don't update the novacells nodeset here.

for now though I will update this pull request and make the required change in the base job definition too since zuul is complaining about the missing nodeset

Reply:
LGTM